src/routes/(dashboard)/ManagerDashboard.svelte

```diff
@@ -502,14 +504,17 @@
     isTransacting={isAssigning}
     primaryButtonText={'Assign'}
     primaryButtonOnClick={() => updateContent(assignEditor)}
-    primaryButtonDisabled={!assignToUserId}
+    primaryButtonDisabled={!assignToEditorUserId}
     bind:open={isAssignContentModalOpen}
-    header={'Choose a user'}
+    header={'Assign Resource(s)'}
 >
+    <h3 class="my-4 text-xl">Editor<span class="text-error">*</span></h3>
+    <UserSelector users={data.users ?? []} defaultLabel="Select Editor" bind:selectedUserId={assignToEditorUserId} />
```

Comment:
Do we still want the restriction to only show editors and managers on this one? And I guess reviewers too now?

Reply:
The ticket called for "all users in company" in the `Editor` dropdown  and reviewers + managers in the `Reviewers` dropdown. If we want different, I'm happy to change it

Reply:
We may want to restrict to prevent report viewers from being selected, though I'm not sure which companies they're in so may be a moot point.

Reply:
@jwinston-bn what do you think?

Reply:
Report viewers can technically be in any company, so we shouldn't be including them in any selectors. That said, there are only 3, two of them on BiblioNexus side, and 1 in their own company.